packages/deployment/upgrade-test/Dockerfile

```diff
@@ -9,96 +13,111 @@ RUN echo "${BOOTSTRAP_MODE}"
 RUN mkdir -p /usr/src/agoric-sdk/upgrade-test-scripts
 WORKDIR /usr/src/agoric-sdk/
 COPY ./start_ag0.sh ./upgrade-test-scripts/
-COPY ./bash_entrypoint.sh ./env_setup.sh ./start_to_to.sh ./upgrade-test-scripts/
+COPY ./bash_entrypoint.sh ./env_setup.sh ./start_to_to.sh ./package.json ./*.mjs ./upgrade-test-scripts/
```

Comment:
why copy these extra files?

Reply:
Looks like just to make all the different targets consistent.  However, I disagree with this... the copy should be absolutely minimal for each target, or the build cache is invalidated more often than it should be!